### ISSUE TITLE
Error after using the nitro

### ISSUE THREAD

**Issue by ArtysV** [KrnkTV/qb-nitro#3]
![image](https://user-images.githubusercontent.com/98827458/172259076-c2cb3a3d-e74f-47bc-aa22-397e8f305686.png)

**Comment by KrnkTV** [comment]
Try new version

**Comment by KrnkTV** [comment]
Left ctrl is for nitro left shift is for changing to purge and pageup and pagedown is for changing purge flow rate

**Comment by divinity-rp** [comment]
yes im trying that but i can change with shift between nitro mode and purge, but and with page up and down works, but if i press ctrl to use it , nothing happens. 

I stopped qb-tunerchip and upgraded the car but still doent work

**Comment by divinity-rp** [comment]
used the latest one on your repo

**Comment by KrnkTV** [comment]
Any errors ? Because ita working perfect with me

**Comment by divinity-rp** [comment]
no errors at all - going to restart server and clear cache and check again - will let you know now - thank you for the response.

**Comment by divinity-rp** [comment]
still no errors and not working for us

**Comment by divinity-rp** [comment]
thank you so much

**Comment by divinity-rp** [comment]
any news perhaps?

**Comment by KrnkTV** [comment]
Its working perfect with me idk why dont want to work with you
